pixi/README.md

```diff
@@ -0,0 +1,32 @@
+# AMP by Example - Playground
```

Comment:
For now, let's replace this text with brief instructions for building the Pixi frontend.

Reply:
Created a rough readme.